mesheryctl/pkg/utils/helpers.go

```diff
@@ -770,3 +772,10 @@ func StringContainedInSlice(str string, slice []string) int {
 	}
 	return -1
 }
+
+func CreateDefaultSpinner(suffix string, finalMsg string) *spinner.Spinner {
+	s := spinner.New(spinner.CharSets[11], 100*time.Millisecond)
```

Comment:
Is the duration of this spinner statically configured (meaning that it will be the same no matter how long the pending process takes)?

Reply:
I think this is the last question of the PR.

Reply:
@leecalcote The time mentioned there is the speed of "animation" of the spinner.

The spinner will keep spinning until an operation is complete.

Basically-

1. start spinner
2. do something
3. stop spinner

The spinner will stop and give out a log after "something" is done.

Reply:
This PR needs to be changed, I'm making the changes currently and then we can review